jsk_apc2016_common/plugin.xml

```diff
@@ -0,0 +1,15 @@
+<library path="src">
+  <class name="SelectTarget" type="jsk_apc2016_common.rqt_plugin.select_target.SelectTarget" base_class_type="rqt_gui_py::Plugin">
+    <description>Select target object for each bin.</description>
+    <qtgui>
+      <group>
+        <label>Visualization</label>
+        <icon type="theme">folder</icon>
+        <statustip>Plugins related to visualization.</statustip>
```

Comment:
This is very similar to rqt_reconfigure, so why not follow rqt_reconfigure?
https://github.com/ros-visualization/rqt_common_plugins/blob/master/rqt_reconfigure/plugin.xml#L8